en/docs/chapter_hashing/hash_map.md

```diff
@@ -1,30 +1,30 @@
 # Hash Table
 
-A "hash table", also known as a "hash map", achieves efficient element querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can retrieve the corresponding `value` in $O(1)$ time.
+*Hash Table*, also known as *Hash Map*, achieves efficient querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can get the corresponding `value` in $O(1)$ time complexity.
 
-As shown in the figure below, given $n$ students, each with two pieces of data: "name" and "student number". If we want to implement a query feature that returns the corresponding name when given a student number, we can use the hash table shown in the figure below.
+As figure shows below, given $n$ students, each with two attributes: "Name" and "Student ID". If we want to implement a query that returns the corresponding name of the given student ID, we can use the hash table shown in the figure below.
```

Comment:
"As shown in the figure below," is generally considered a more formal and clearer way of introducing a figure in academic or professional writing.